threescale_api/utils.py

```diff
@@ -23,7 +23,7 @@ def extract_response(response: requests.Response, entity: str = None,
     if collection and collection in extracted:
         extracted = extracted.get(collection)
     if isinstance(extracted, list):
-        return [value.get(entity) for value in extracted]
+        return [value.get(entity) for value in extracted if entity in value]
```

Comment:
I don't like that we have two clients for the same thing: building and non-buildin'.
With that in mind, my suggestion is to allow the user to define multiple `entity_name` in a form of a list of strings. Possible solution:
```
if isinstance(extracted, list):
    if isinstance(entity, str):
        return [value.get(entity) for value in extracted]
    else:
        return [value for key,value in extracted.items() if k in [entity]]
```